package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "rv-026webui",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "devserver": "node ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --debug --hot --devtool eval-source-map --output-pathinfo --watch --colors --inline --content-base ./ --port 8090 --host 0.0.0.0",
```

Comment:
migrate to gulp

Reply:
@python4eg  Migration to gulp [done](https://github.com/Social-projects-Rivne/Rv-026WebUI/pull/18)!